Formula/curl.rb

```diff
@@ -4,7 +4,6 @@ class Curl < Formula
   url "https://curl.se/download/curl-7.83.1.tar.bz2"
   mirror "https://github.com/curl/curl/releases/download/curl-7_83_1/curl-7.83.1.tar.bz2"
   mirror "http://fresh-center.net/linux/www/curl-7.83.1.tar.bz2"
-  mirror "http://fresh-center.net/linux/www/legacy/curl-7.83.1.tar.bz2"
```

Comment:
This mirror becomes valid during the period of time between the next version being released and we update the formula for the new version.

For example:
http://fresh-center.net/linux/www/curl-7.83.0.tar.bz2 is gone
http://fresh-center.net/linux/www/legacy/curl-7.83.0.tar.bz2 is active